Shared.NFe.Classes/Informacoes/Detalhe/ProdEspecifico/veicProd.cs

```diff
@@ -72,20 +68,12 @@ public class veicProd : ProdutoEspecifico
         /// <summary>
         ///     J08 - Peso Líquido
         /// </summary>
-        public decimal pesoL
-        {
-            get { return _pesoL; }
-            set { _pesoL = value.Arredondar(3); }
-        }
+        public string pesoL { get; set; }
 
         /// <summary>
         ///     J09 - Peso Bruto
         /// </summary>
-        public decimal pesoB
-        {
-            get { return _pesoB; }
-            set { _pesoB = value.Arredondar(3); }
-        }
+        public decimal pesoB { get; set; }
```

Comment:
Este também precisar ser tipo string.
```suggestion
        public string pesoB { get; set; }
```

Reply:
Feito